app/build.gradle

```
@@ -29,13 +29,29 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    buildFeatures {
+        viewBinding true
+    }
+    dataBinding {
+        enabled = true
+    }
```

```suggestion
    buildFeatures {
        viewBinding true
        dataBinding true
    }
```

최근 gradle에서는 이렇게도 사용할 수 있습니다.

---

최곱니다 썬생님!